source/Halibut/Transport/Streams/ReadIntoMemoryBufferStream.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Halibut.Transport.Observability;
+
+namespace Halibut.Transport.Streams
+{
+    public class ReadIntoMemoryBufferStream : Stream
+    {
+        readonly MemoryStream memoryStream;
+        readonly Stream sourceStream;
+        readonly long readIntoMemoryLimitBytes;
+        readonly OnDispose onDispose;
+
+        bool limitReached;
```

Comment:
I deleted this field and the `ReadIntoMemoryBufferStreamFixture` passed.

is it needed?

![Uploading image.png…]()

Reply:
Huh, yeah, it isn't needed. Removed!